packages/react/src/components/SuiteHeader/util/IdleTimer.js

```diff
@@ -39,8 +39,9 @@ class IdleTimer {
     }
     // Initialize the setInterval thread
     this.intervalHandler = setInterval(() => {
-      // Check if user is idle
-      if (this.getUserInactivityTimeout() < Date.now()) {
+      // Check if user is idle or if userInactivityTimeout is not a number, which indicates that a logout has already happened in another tab
+      const userInactivityTimeout = this.getUserInactivityTimeout();
+      if (Number.isNaN(userInactivityTimeout) || userInactivityTimeout < Date.now()) {
```

Comment:
nit: we could use well named variables to explain what is going on, they normally stand the test of time better than comments.

```
const userInactivityTimeoutValue = this.getUserInactivityTimeout();
const userIsIdle = userInactivityTimeoutValue < Date.now();
const userLoggedOutInDifferentTab = Number.isNaN(userInactivityTimeoutValue);
if (userIsIdle || userLoggedOutInDifferentTab) {
...
```

Reply:
Thanks, good idea, I'll make the changes (but `userLoggedOutInDifferentTab` comes first in the `||` logic)